src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/create-video-unit/create-video-unit.component.html

```diff
@@ -0,0 +1,9 @@
+<div *ngIf="isLoading" class="d-flex justify-content-center">
+    <div class="spinner-border" role="status">
+        <span class="sr-only">Loading...</span>
+    </div>
+</div>
+
+<div *ngIf="!isLoading" class="container">
+    <jhi-video-unit-form [isEditMode]="false" (formSubmitted)="createAttachmentUnit($event)"></jhi-video-unit-form>
```

Comment:
```suggestion
    <jhi-video-unit-form [isEditMode]="false" (formSubmitted)="createVideoUnit($event)"></jhi-video-unit-form>
```

Reply:
Done in b2ad6f08755d798fe85237719dc8f1e8de51e93e